src/mailing/sender.js

```diff
@@ -52,6 +52,27 @@ async function sendOrderCompletedEmail(to, username, orderItems, total) {
   });
 }
 
+// Function to send the welcome email
+async function sendWelcomeEmail(recipient, user) {
+  const template = 'welcome';
+  const locals = { user };
+  try {
+    const result = await baseEmail.send({
+      template,
+      message: {
+        to: recipient,
+      },
+      locals,
+    });
+    console.log(result);
```

Comment:
We can probably remove this before merging

Reply:
console.log removed, commit 335d0c2b8c8a1d490beab585ecd607a7a65997a9